about.html

```diff
@@ -0,0 +1,239 @@
+                        <li>
+                            <a href="./index.html">Home</a>
+                        </li>
+                        <li>
+                            <a href="#">Program</a>
+                        </li>
+                        <li>
+                            <a href="#">Join</a>
+                        </li>
+                        <li>
+                            <a href="#">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="#">News</a>
+                        </li>
+                        <li class="cc_campaign">
+                            <a href="#">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+    </header>
+
+    <section class="headline_wrapper">
+        <div class="headline">
+            <div class="title_wrapper">
+                <div class="suport_title hide--desktop">
+                    <p>"Hello! Sharing world"</p>
+                </div>
+                <div class="title">
+                    <h1>
+                        CREATIVE CODERS GLOBAL SUMMIT 2023
+                    </h1>
+                </div>
+            </div>
+        </div>
+
+        <div class="description_wrapper">
+            <div class="description">
+                <p class="hide--desktop">The Global Summit brings together the community of
+                    experts, academics, and activists who comprise the Creative Commons affiliate network in a different
+                    country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals.</p>
+
+                <p class="hide--mobile">A developer conference is a gathering of software developers, engineers, and
+                    other professionals who work in the field of software development. These conferences are typically
+                    held in large cities around the world and are organized by companies, professional organizations, or
+                    communities of developers. The main goal of a developer conference is to provide a forum for
+                    developers to learn about new technologies, techniques, and best practices in software development.
+                    At a developer conference, attendees can expect to participate in a variety of activities such as
+                    presentations, workshops, panels, and networking events. These events are typically led by industry
+                    experts and thought leaders, who share their knowledge and experience with the attendees. Many
+                    developer conferences also feature exhibits and demos of new products and tools, giving attendees
+                    the opportunity to try out the latest technology firsthand.</p>
+            </div>
+        </div>
```

Comment:
- [OPTIONAL]  The content of your `<h1>` tag overlaps the content of your `<p>` tag both on the desktop and mobile versions. Kindly look into it and fix it. Take a look :point_down: 

### Desktop
![your-about-page-desktop](https://user-images.githubusercontent.com/26597432/210701551-561b9939-0707-4348-8413-11d78931737e.png)

### Mobile
![your-about-mobile](https://user-images.githubusercontent.com/26597432/210701584-1eb2b1a9-9384-4269-b556-babdebb6e27d.png)

### Guiding Template Desktop
![temp-about-desktop](https://user-images.githubusercontent.com/26597432/210702882-f24951fe-94f9-4663-a698-4a56dc7affee.png)

### Guiding Template Mobile
![temp-about-mobile](https://user-images.githubusercontent.com/26597432/210702906-dbb21294-6468-4b6a-a352-06c37b98c421.png)